docs/CaliptraIntegrationSpecification.md

```diff
@@ -67,6 +67,8 @@ The following table describes integration parameters.
 | CALIPTRA_INTERNAL_QSPI  | config_defines.svh | Defining this enables the internal QSPI.        |
 | USER_ICG                | config_defines.svh | If added by an integrator, provides the name of the custom clock gating module that will be used in [clk_gate.sv](../src/libs/rtl/clk_gate.sv). This substitution is only performed if integrators also define TECH_SPECIFIC_ICG. |
 | TECH_SPECIFIC_ICG       | config_defines.svh | Defining this causes the custom, integrator-defined clock gate module (indicated by the macro USER_ICG) to be used in place of the native Caliptra clock gate module. |
+| USER_EC_RV_ICG          | config_defines.svh | If added by an integrator, provides the name of the custom clock gating module that will be used in the RISC-V core. This replaces the clock gating module TEC_RV_ICG defined in [beh_lib.sv](../src/riscv_core/veer_el2/rtl/lib/beh_lib.sv). This substitution is only performed if integrators also define TECH_SPECIFIC_EC_RV_ICG. |
```

Comment:
- Future tense needed here? If not, `will be used in the RISC-V core` could change to `is used in the RISC-V core`
- Recommend minor updates to second sentence: `USER_EC_RV_ICG replaces the clock gating module, TEC_RV_ICG, defined in...`

Reply:
Thanks, fixed